models/intermediate/int_pendo__page_info.sql

```diff
@@ -0,0 +1,70 @@
+with page as (
+
+    select *
+    from {{ ref('int_pendo__latest_page') }}
+),
+
+application as (
+
+    select *
+    from {{ ref('int_pendo__latest_application') }}
+),
+
+pendo_user as (
+
+    select *
+    from {{ var('user') }}
+),
+
+product_area as (
+
+    select *
+    from {{ var('group') }}
+),
+
+page_rule as (
+
+    select *
+    from {{ ref('int_pendo__latest_page_rule') }}
+),
+
+agg_page_rule as (
+
+    select 
+        page_id,
+        -- should we use a different/more apparent delimiter?
+        {{ fivetran_utils.string_agg( "rule", "', '" ) }} as rules 
```

Comment:
My only other thought would be a tilde `~` as I have a hard time thinking this would ever be in a rule, but I also don't see any commas in our rules. For consistency with our other packages I say we keep the comma and space as I don't see this being an issue.